### DISCUSSION TITLE
Seconion 2.3.2 Release Notes on airgap

### DISCUSSION THREAD

**Discussion by threathunternotebook** [Security-Onion-Solutions/securityonion#1870] [Unsupported Versions] [answered]
The release notes state the following:
"Cheat sheet is now available for airgap installs."
Where would I find this cheat sheet?

**Comment by shipler** [comment]
1. https://master-ip/docs
2. In the SOC (master web application) click on the person img on the top right corner, press help

**Comment by threathunternotebook** [comment, marked as answer]
Sorry, this was a misunderstanding.  I thought this was a cheatsheet that
explained the airgap installation process in more detail.  What you have is
the Security Onion Cheat Sheet installed locally on the master/manager.
